eq-author/src/constants/unit-types.js

```diff
@@ -0,0 +1,63 @@
+export const CENTIMETERS = "Centimeters";
+export const METERS = "Meters";
+export const KILOMETERS = "Kilometers";
+export const MILES = "Miles";
+export const SQUARE_CENTIMETERS = "Square centimeters";
+export const SQUARE_METERS = "Square meters";
+export const SQUARE_KILOMETERS = "Square kilometers";
+export const SQUARE_MILES = "Square miles";
+export const ACRE = "Acre";
+export const HECTARE = "Hectare";
+export const CUBIC_CENTIMETERS = "Cubic centimeters";
+export const CUBIC_METERS = "Cubic meters";
+export const CUBIC_KILOMETERS = "Cubic kilometers";
+export const TONNES = "Tonnes";
+export const KILOJOULES = "Kilojoules";
+export const KILOWATT_HOUR = "Kilowatt hour";
+
+export const unitConversion = {
+  [CENTIMETERS]: { unit: CENTIMETERS, type: "Length", abbreviation: "cm" },
+  [METERS]: { unit: METERS, type: "Length", abbreviation: "m" },
+  [KILOMETERS]: { unit: KILOMETERS, type: "Length", abbreviation: "km" },
+  [MILES]: { unit: MILES, type: "Length", abbreviation: "mi" },
+  [SQUARE_CENTIMETERS]: {
+    unit: SQUARE_CENTIMETERS,
+    type: "Area",
+    abbreviation: `cm${String.fromCharCode(178)}`,
```

Comment:
As discussed this can be the actual character pasted into the file as it is all utf8 compatible